index.d.ts

```diff
@@ -1039,6 +1039,9 @@ declare module 'plaid' {
       cb: Callback<TransactionsAllResponse>,
     ): void;
 
+    // refreshTransactions(String, Function)
+    refreshTransactions: AccessTokenFn<BaseResponse>;
```

Comment:
I just want to confirm that using `BaseResponse` here was intentional

Reply:
Yes, the response (currently) only includes the `request_id`. For context, the client would then have to call `/transactions/get` to retrieve transactions following this synchronous request. 

If you think it would be better, I can create an empty interface that extends `BaseResponse`

Reply:
nope this is fine! 👍